### PR TITLE
chore: fix `v4` migration `TypeError: Cannot read properties of null (reading 'getSourceFiles')`

### PR DESCRIPTION
**Pull request by nsbarsukov**
Relates to #5808

# Previous behaviour

```shell
➜ schematics ./dist/cdk:updateToV4 --allow-private --dry-run true
Debug mode enabled by default for local collections.
 

🚀 Your packages will be updated to @taiga-ui/*@^3.59.0

  ⚡️ replacing identifiers...
An error occured:
TypeError: Cannot read properties of null (reading 'getSourceFiles')
    at getSourceFiles (/Users/n.barsukov/WebstormProjects/taiga-ui/node_modules/ng-morph/source-file/helpers/get-source-files.js:6:45)
    at /Users/n.barsukov/WebstormProjects/taiga-ui/node_modules/ng-morph/imports/helpers/get-imports.js:7:58
    at getDeclaration (/Users/n.barsukov/WebstormProjects/taiga-ui/node_modules/ng-morph/utils/helpers/get-declaration-getter.js:7:16)
    at getNamedImportReferences (/Users/n.barsukov/WebstormProjects/taiga-ui/dist/cdk/schematics/utils/get-named-import-references.js:7:58)
    at replaceIdentifier (/Users/n.barsukov/WebstormProjects/taiga-ui/dist/cdk/schematics/ng-update/steps/replace-identifier.js:18:83)
    at Array.forEach (<anonymous>)
    at replaceIdentifiers (/Users/n.barsukov/WebstormProjects/taiga-ui/dist/cdk/schematics/ng-update/steps/replace-identifier.js:12:15)
    at /Users/n.barsukov/WebstormProjects/taiga-ui/dist/cdk/schematics/ng-update/v4/index.js:19:40
    at callRuleAsync (/Users/n.barsukov/WebstormProjects/taiga-ui/node_modules/@angular-devkit/schematics/src/rules/call.js:77:24)
    at /Users/n.barsukov/WebstormProjects/taiga-ui/node_modules/@angular-devkit/schematics/src/rules/call.js:72:40
```

# New behaviour
```shell
➜ schematics ./dist/cdk:updateToV4 --allow-private --dry-run true

🚀 Your packages will be updated to @taiga-ui/*@^3.59.0
  ⚡️ replacing identifiers...
  ✅  identifiers replaced 

  ⚡️ removing modules...
  ✅  modules removed 

  ⚡️ updating TuiMapper typing to the typed version
  ⚡️ renaming TuiTypedMapper to TuiMapper
 🏆 successfully migrated 
  ⚡️ updating TuiMatcher typing to the typed version
  ⚡️ renaming TuiTypedMatcher to TuiMatcher
 🏆 successfully migrated 
  ⚡️ migrating legacy mask utils...
  ⚡️ tuiMaskedMoneyValueIsEmpty
  ⚡️ tuiMaskedNumberStringToNumber
 🏆 successfully migrated 
  ⚡️ migrating TuiDestroyService => takeUntilDestroyed ...
 🏆 successfully migrated 
  ⚡️ migrating templates...
    (1162 / 1162)  ✅ 
  ✅  templates migrated 

 🏆 We migrated packages to @taiga-ui/*@^3.59.0 in 309.55 sec. 
UPDATE projects/addon-doc/components/documentation/documentation.template.html (9537 bytes)
UPDATE projects/kit/components/checkbox-labeled/checkbox-labeled.template.html (613 bytes)
UPDATE projects/kit/components/filter/filter.template.html (730 bytes)
# [...]
```

+ add CI checks

### CODE REVIEW

File: .github/workflows/schematics.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      - uses: taiga-family/ci/actions/setup/variables@v1.51.14
+      - uses: taiga-family/ci/actions/setup/node@v1.51.14
+
+      - run: npx nx build cdk
+
+      - name: Upload cache / ${{ env.CACHE_DIST_KEY }}
+        uses: actions/cache/save@v4.0.1
+        with:
+          path: dist/cdk
+          key: ${{ env.CACHE_DIST_KEY }}
```
Comment (splincode) [inline .github/workflows/schematics.yml:23]: `key: ${{ env.CACHE_DIST_KEY }}_cdk`

File: .github/workflows/schematics.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    steps:
+      - uses: taiga-family/ci/actions/setup/checkout@v1.51.14
+      - uses: taiga-family/ci/actions/setup/variables@v1.51.14
+      - uses: taiga-family/ci/actions/setup/node@v1.51.14
+
+      - name: Download cache / ${{ env.CACHE_DIST_KEY }}
+        uses: actions/cache/restore@v4.0.1
+        with:
+          path: dist/cdk
+          key: ${{ env.CACHE_DIST_KEY }}
```
Comment (splincode) [inline .github/workflows/schematics.yml:43]: `key: ${{ env.CACHE_DIST_KEY }}_cdk`

File: .github/workflows/schematics.yml (hunk 1 above)
Comment (nsbarsukov) [reply .github/workflows/schematics.yml]: Fixed in 5d3df851a88dcda8ed3a9d378acc16d280c654ae

File: .github/workflows/schematics.yml (hunk 2 above)
Comment (nsbarsukov) [reply .github/workflows/schematics.yml]: Fixed in 5d3df851a88dcda8ed3a9d378acc16d280c654ae

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**